frontend/app/work_packages/controllers/work-packages-list-controller.js

```diff
@@ -287,4 +287,15 @@ module.exports = function($scope, $rootScope, $state, $location, latestTab,
       );
     }
   };
+
+  $scope.getFilterCount = function() {
```

Comment:
@0xF013 I think it's ok to bind to function, when result shouldn't be expensive. WDYT? Cleaner than creating watcher + updating a scalar property on scope.